app/controllers/editions_controller.rb

```diff
@@ -331,6 +347,10 @@ def description(r)
 
 private
 
+  def redirect_url
+    params.fetch("redirect_url", "").sub(%r{^https?://(www\.)?gov\.uk/}, "/")
```

Comment:
This has now been broken out in to two methods, which hopefully should be a bit clearer 👍